app/models/parsers/cardinal_pbmc_count_parser.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+module Parsers
+  # A parser for the cardinal pipeline qc file
+  class CardinalPbmcCountParser
+    class_attribute :assay_type, :assay_version
+
+    HEADERS = [
+      'Well Name',
+      'Live Count',
+      'Live Cells/mL',
+      'Live Mean Size',
+      'Viability',
+      'Dead Count',
+      'Dead Cells/mL',
+      'Dead Mean Size',
+      'Total Count',
+      'Total Cells/mL',
+      'Total Mean Size',
+      'Note:',
+      'Errors:'
+    ].freeze
+
+    self.assay_type = 'Cardinal_PBMC_Count'
+    self.assay_version = 'v1.0'
+
+    def self.parses?(content)
+      content.split('\r\n')[0][0] == HEADERS
```

Comment:
Worth commenting about the nested arrays that are incoming?  I assume that's why there are two `[0]` here?

Reply:
This is more about how the data is formatted rather than nested arrays for Cardinal. The original file is parsed as a csv which creates a 2 dimensional array.